src/advanced/aleo-proof-system/understanding-proof-systems.md

```diff
@@ -0,0 +1,20 @@
+# Understanding proof systems
+
+Most proof system can be described in three mayor phases: **Preprocessing, Proving and Verification**. Let’s take a look at each phase:
```

Comment:
“mayor” → “major” [ Fix Typo ]

Reply:
Ooups... my Spanish is showing up hehe..
Fixed...